subprojects/docs/src/docs/userguide/jvm/jacoco_report_aggregation_plugin.adoc

```diff
@@ -18,6 +18,13 @@
 The JaCoCo Report Aggregation plugin (plugin id: `jacoco-report-aggregation`) provides the ability to aggregate the results of multiple JaCoCo code coverage reports into a single HTML report.  The binary data backing the coverage reports are produced by link:{groovyDslPath}/org.gradle.api.tasks.testing.Test.html[Test] task invocations; see more at the <<jacoco_plugin#jacoco_plugin,JaCoCo Plugin>> chapter. This aggregation is done in a way that is compatible with the <<configuration_cache#config_cache,configuration cache>>.
```

Comment:
Instead of saying here that this plugin is compatible with the configuration cache (we don't do this anywhere), it should be added to the compatibility matrix in https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:plugins:core

Reply:
Pushed - though as "new" core plugins, they don't have a explicit issue for implementing configuration cache compatibiltiy.